src/components/fields/DataSelector.js

```diff
@@ -24,18 +24,10 @@ export class UnconnectedDataSelector extends Component {
   setLocals(props, context) {
     this.dataSources = context.dataSources || {};
     this.dataSourceOptions = context.dataSourceOptions || [];
-    this.dataSrcExists = false;
-    if (attributeIsData(props.attrMeta)) {
-      this.dataSrcExists = true;
-      this.srcAttr = props.attr + 'src';
-      this.srcProperty = nestedProperty(props.container, this.srcAttr);
-    }
 
-    if (this.dataSrcExists) {
-      this.fullValue = this.srcProperty.get();
-    } else {
-      this.fullValue = this.props.fullValue;
-    }
+    this.srcAttr = props.attr + 'src';
```

Comment:
@bpostlethwaite please think on this part of the change... I'm not sure why the `if` was there before but I need to be able to write to `srcAttr`  :)

Reply:
Sorry @nicolaskruchten I can't remember for the life of me. It could have been necessary in some permutation of DataSelector but become unnecessary at some point. 

We'll probably figure out why in a couple weeks...